src/shell/meanbee/configpoweredcss/regenerate.php

```diff
@@ -0,0 +1,48 @@
+<?php
+require_once __DIR__ . '/../../abstract.php';
+
+class Meanbee_Configpoweredcss_Regenerate extends Mage_Shell_Abstract
+{
+
+    /**
+     * Run script
+     *
+     */
+    public function run()
+    {
+
+        /**
+         * Shell script regenerate the configpowered css for all stores
+         */
+
+        try {
+
+            /** @var Meanbee_ConfigPoweredCss_Model_Css $css */
+            $css = Mage::getModel('meanbee_configpoweredcss/css');
+
+            $stores = Mage::app()->getStores();
+            foreach ($stores as $storeId => $store) {
+                try {
+                    Mage::app()->setCurrentStore($storeId);
+                    $css->publish($storeId);
+                } catch (Exception $e) {
+                    Mage::logException($e);
+                    return;
+                }
+            }
+
+            fwrite(STDERR,
+                "Config powered css has been regenerated successfully.\n");
+            exit(0);
+        } catch (Exception $e) {
```

Comment:
What exceptions are we trying to catch here? This seems unnecessary.

Reply:
I'd rather catch an exception then break a build process, as this is only generating css.

Reply:
This is wrong on several levels:

1. Silently ignoring exceptions is bad practice. The deployment will go green and we'll happily move along only to discover later that the site had problems. If there are non-breaking exceptions, we can catch and alert, but catching **all** exceptions and ignoring them is just bad code.
2. You have nested try-catch blocks with different behaviours. The actual "meat" of the work is wrapped in a try-catch which exits with an error code, so if something breaks with the CSS generation, we are actually going to stop the deployment. This try-catch block really only covers lines 21-24, where either we don't have the model we expect (bad, build breaking) or can't fetch stores (bad, build breaking). So again, what ignorable exceptions are we expecting?